packages/typespec-ts/src/modular/emitModels.ts

```diff
@@ -84,7 +90,7 @@ function buildModelInterface(
 ): InterfaceStructure {
   const modelProperties = model.properties ?? [];
   const modelInterface = {
-    name: model.name ?? "FIXMYNAME",
+    name: model.alias ?? model.name ?? "FIXMYNAME",
```

Comment:
Could you add some UT for XXXParent type in https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/test/modularUnit/typeHelpers.spec.ts#L7?

I am a little bit unclear what the type is?